appservice/src/intTest/groovy/nu/yona/server/AbstractAppServiceIntegrationTest.groovy

```diff
@@ -95,6 +95,7 @@ abstract class AbstractAppServiceIntegrationTest extends Specification
 	{
 		def richard = addRichard(false)
 		def bob = addBob(false)
+		bob.emailAddress = "bob@dunn.com"
```

Comment:
It's attractive indeed, but cumbersome and error prone. In general, a user does not have an e-mail address, which means that we loose it upon lots of actions: reload, update, number confirmation, etc.

I tried it out, but that won't make things simpler.